src/screens/Home.js

```diff
@@ -1,6 +1,8 @@
 import { render } from '../shared/dom.js';
-import { MovieCarousel } from '../components';
+import { MovieTile } from '../components';
 import ApiProvider from '../providers/ApiProvider.js';
```

Comment:
Approved, but please remove these imports which are unnecessary

Reply:
I will approve after corrections :)

Reply:
done in newest commit